python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -51,26 +81,44 @@ python paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
 ```
 The converted data binary file is saved by default in `~/.cache/paddle/dataset/int8/download/int8_full_val.bin`
  * ### Prepare model
-You can run the following commands to download ResNet50 model.
+You can run the following commands to download ResNet50 model. Since for performance optimization, we should be aware of the graph optimization during the QAT training phase, so we have the different fake QAT models between QAT MKL-DNN 1.0 and QAT MKL-DNN 2.0.
 
 ```bash
 mkdir -p /PATH/TO/DOWNLOAD/MODEL/
 cd /PATH/TO/DOWNLOAD/MODEL/
 export MODEL_NAME=ResNet50
+# QAT 1.0
 wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${MODEL_NAME}_qat_model.tar.gz
+# QAT 2.0
+wget http://paddle-inference-dist.bj.bcebos.com/int8/QAT_models/${MODEL_NAME}_qat_perf.tar.gz
 mkdir -p ${MODEL_NAME}
 tar -xvf ${MODEL_NAME}_qat_model.tar.gz -C ${MODEL_NAME}
 ```
 
 To download and verify all the 7 models, you need to set `MODEL_NAME` to one of the following values in command line:
-
 ```text
+QAT MKL-DNN 1.0
 MODEL_NAME=ResNet50, ResNet101, GoogleNet, MobileNetV1, MobileNetV2, VGG16, VGG19
+QAT MKL-DNN 2.0
+MODEL_NAME=resnet50, resnet101, mobilenetv1, mobilenetv2, vgg16, vgg19 
 ```
 * ### Commands to reproduce benchmark
-You can run `qat_int8_comparison.py` with the following arguments to reproduce the accuracy result on ResNet50.
+You can run `qat_int8_comparison.py` with the following arguments to reproduce the accuracy result on ResNet50. The difference of command line between the QAT MKL-DNN 1.0 and QAT MKL-DNN 2.0 is that we use argument `qat2` to enable QAT MKL-DNN 2.0. Besides, since QAT MKL-DNN 2.0 needs the performance test, we use `OMP_NUM_THREADS=1` and `batch_size=1` in QAT MKL-DNN 2.0.
```

Comment:
```
Besides, since QAT MKL-DNN 2.0 needs the performance test, we use `OMP_NUM_THREADS=1` and `batch_size=1` in QAT MKL-DNN 2.0.
```
->
```
`To perform QAT MKL-DNN 2.0 the performance test, the environmental variable `OMP_NUM_THREADS=1` and `batch_size=1` parameter should be set.
```

Reply:
Done